packages/mesos-modules/build

```diff
@@ -1,15 +1,29 @@
 #!/bin/bash
+set -o nounset -o pipefail
+
 pushd /pkg/src/mesos-modules/
 ./bootstrap
 popd
 
-source /opt/mesosphere/environment.export
+export LD_LIBRARY_PATH=/opt/mesosphere/lib
+
+# Add `protoc` to the path.
+export PATH=/opt/mesosphere/lib/mesos/3rdparty/bin:$PATH
```

Comment:
:thinking: I suppose it is odd to have binaries in the `lib` directory.  Seems reasonable to move into the binary folder.